stubs/regex/regex/_regex_core.pyi

```diff
@@ -44,42 +44,42 @@ class RegexFlag(enum.IntFlag):
     X = 0x40
     VERBOSE = X
 
-A: int
-ASCII: int
-B: int
-BESTMATCH: int
-D: int
-DEBUG: int
-E: int
-ENHANCEMATCH: int
-F: int
-FULLCASE: int
-I: int
-IGNORECASE: int
-L: int
-LOCALE: int
-M: int
-MULTILINE: int
-P: int
-POSIX: int
-R: int
-REVERSE: int
-T: int
-TEMPLATE: int
-S: int
-DOTALL: int
-U: int
-UNICODE: int
-V0: int
-VERSION0: int
-V1: int
-VERSION1: int
-W: int
-WORD: int
-X: int
-VERBOSE: int
+A = RegexFlag.A
```

Comment:
Should all of the module constants in this file be `Final`?

```suggestion
A: Final = RegexFlag.A
```

Reply:
Good idea, done.

Reply:
Looks like pytype isn't happy about this change

Reply:
Ah well, thanks for trying! :-)